src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -246,6 +351,10 @@ def process_nic_create_namespace(namespace):
     if not namespace.network_security_group:
         namespace.network_security_group_type = 'none'
 
+def process_nic_namespace(namespace):
```

Comment:
Nope. I think it became process_nic_create_namespace and then stuck around due to a wonky merge.

---
In reply to: [73620111](https://github.com/Azure/azure-cli/pull/616#discussion_r73620111) [](ancestors = 73620111)